jetstream/core/utils/async_multifuture.py

```diff
@@ -0,0 +1,89 @@
+import asyncio
```

Comment:
Where is this module used? I can't find any references.

Reply:
In orchestrator, the `return_channel`, https://github.com/google/JetStream/pull/13/files/09e0e659ba821e14c5595dbfb3c8ec620725d865#diff-753df909960695fd6aa61b95c4762c4edc50a298007d40de7096a7e49e0e1215R130